### PR TITLE
Rename varsMap to argsMap

### PR DESCRIPTION
**Pull request by jgwhite**
Previously this option worked with variable names rather than argument names. We believe mapping argument names is more useful and more likely to be the expected behavior. This patch removes any notion of mapping variable names and focuses strictly on argument names whilst also renaming the option to make it clearer what is being mapped.

### CODE REVIEW

File: addon/mocks/query.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ import { partial } from '../utils';
 
 export const composeMockQuery =
   (getTypeForField, getFieldNameAndAlias, createFieldInfo, resolveFieldInfo, getIsInterface, logError) =>
-    (db, options, _, vars, __, { fieldNodes, fragments, returnType, schema }) => {
+    (db, options, _, vars, __, info) => {
```
Comment (jneurock) [inline addon/mocks/query.js]: Not a required change: Is the `vars` argument used now? If not, maybe we should change this function signature so it has even more underscores?

Also, could you tell me what `info.variableValues` looks like? I haven't seen this before and I'm just curious.

Comment (jgwhite) [reply addon/mocks/query.js]: `variableValues` is the variables as they come over the wire in the request body

Comment (jgwhite) [reply addon/mocks/query.js]: And yes, I think switching to underscores is a good call

Comment (jgwhite) [reply addon/mocks/query.js]: > maybe we should change this function signature so it has even more underscores

Done 👍

**Pull request merged**